Doc/library/dataclasses.rst

```diff
@@ -264,6 +274,16 @@ Module contents
    - *repr*: If true (the default), this field is included in the
      string returned by the generated :meth:`~object.__repr__` method.
 
+     .. doctest::
+
+        >>> @dataclass
+        ... class InventoryItem:
+        ...     name: str
+        ...     unit_price: float
+        ...     quantity_on_hand: int = field(repr=False)
+        >>> InventoryItem(name='widget', unit_price=3.0, quantity_on_hand=10)
```

Comment:
```suggestion
        ...     quantity_on_hand: int = field(repr=False)
        ...
        >>> InventoryItem(name='widget', unit_price=3.0, quantity_on_hand=10)
```

We could also decide to have another example where `repr=False` would make more sense but otherwise I think it's fine.